src/index.js

```diff
@@ -1,12 +1,15 @@
 const express = require("express")
 const cors = require("cors")
+const authRoute = require("../routes/authRoute")
 
 const PORT = process.env.PORT || 8000
 
 const app = express()
 app.use(cors())
 app.use(express.json())
 
+app.use("/auth", authRoute)
+
 app.listen(PORT, (err) => {
   if (err) {
     console.log(`ERROR: ${err}`)
```

Comment:
Tambahin ini buat informasi aja

```suggestion
app.listen(PORT, (err) => {
  console.log(`Server running on Port ${PORT})
  if (err) {
    console.log(`ERROR: ${err}`)
```

Reply:
ohh ini di rubah jadi yg ijo kan?

Reply:
yup